dashboard/app/views/pd/workshop_mailer/exit_survey.html.haml

```diff
@@ -13,22 +13,23 @@
   = @workshop.subject
   workshop and for your commitment to bringing foundational 21st century learning to your students!
 
-- if @is_first_workshop
-  %p
-    Computer science has become the fastest-growing subject in K-12 education,
-    thanks to inspirational educators like you who have decided to spend your summer or weekends
-    in professional learning for the benefit of your classroom and the future of our children.
-    Thank you for joining the teacher-powered movement to give every student in every school
-    the opportunity to learn computer science.
-
 %p
   = link_to('Please take a minute to complete this short survey', @survey_url) + '.'
```

Comment:
Also needs this sentence right after "Let us know how we can improve your experience and your learning."